tests/components/arve/test_sensor.py

```diff
@@ -0,0 +1,46 @@
+"""Test for Arve sensors."""
```

Comment:
Hello, thanks for all of your comments and help. I'm now running and fixing the tests and I'm facing the error during `test_form_invalid_auth()` test. The error is : "Error doing job: Unclosed client session". I face this issue also when I'm adding an Arve device to my local homeassistant instance. Here is the screenshot. 
<img width="702" alt="Bildschirmfoto 2024-03-19 um 15 31 54" src="https://github.com/home-assistant/core/assets/146177275/bc586591-7351-4ca4-942d-ddbfab0bb166">
In logs from test I can see that it is my library for this integration causes the issue, when it starts a new client session, but I can't understand why. For each device there should be a session and they shouldn't interfere (which I suppose happens). If you have any suggestions or previous experience with this problem, I would be very grateful if you could help me.

Reply:
I don't really know where this is coming from sorry. Only bit I can suggest is: There is a helper to get the central HA `aiohttp` clientsession `from homeassistant.helpers.aiohttp_client import async_get_clientsession` which you can use to get a clientsession to pass to your lib like this `async_get_clientsession(hass)`. It's something you can try out, but I doubt this will fix the problem tbh

Reply:
What you can also try, is updating your `aiohttp`, maybe you're using an old version

Reply:
Thank you, I will try these steps

Reply:
The strange thing is that however I receive the error above, the device is being added anyways to the local homeassistant instance and gets actual data. The main problem is that I can't pass the test